ChangeLog-10.5.md

```diff
@@ -1,6 +1,6 @@
 # Changes in PHPUnit 10.5
 
-All notable changes of the PHPUnit 10.5 release series are documented in this file using the [Keep a CHANGELOG](https://keepachangelog.com/) principles.
+All notable changes of the PHPUnit 10.5 release series are documented in this file using the [Keep a CHANGELOG](https://keepachangelog.com/) principles
```

Comment:
This is unrelated to the issue at hand.

Reply:
Heya yeah sorry. I needed a way to trigger CI on my fork. I reverted this.